charts/auctioneer/values.yaml

```diff
@@ -9,9 +9,8 @@ images:
   auctioneer:
     repo: ghcr.io/astriaorg/auctioneer
     pullPolicy: IfNotPresent
-    # TODO - update to latest tag
-    tag: "pr-1839"
-    devTag: "pr-1839"
+    tag: sha-2db84ed
```

Comment:
We need to merge https://github.com/astriaorg/astria/pull/1972 for these changes to go into main.